models/image-data.models.ts

```diff
@@ -0,0 +1,36 @@
+import * as path from 'path';
+import { ObjectMetadata } from 'firebase-functions/lib/providers/storage';
+
+export class ImageData {
```

Comment:
Is this intended to replace or extend the `TanamFile` model? I think that a double declaration of similar classes/interfaces that has almost the same usage and purpose must be clarified to be explained where it should be used and why we need another one. A similar scenario to this is [DocumentContext](https://github.com/oddbit/tanam/blob/fe548007d3b567b3326bfa2716d784621b4c2de9/models/dynamic-page.models.ts) and [Document](https://github.com/oddbit/tanam/blob/fe548007d3b567b3326bfa2716d784621b4c2de9/models/document.models.ts). 

Please make class comments explaining the need for a app-wide shared model and when to use it instead of the `TanamFile` model.

Reply:
Changes are in fd29210